### ISSUE TITLE
http and https pages have 2 differents fonts

### ISSUE THREAD

**Issue by Murphinator2000** [jaredhaight/lgod#26]
If you view this page (https) https://www.livegeekordie.com/article/nintendos-super-smashing-sale-week-4/ it has a different font than the one at (http) http://www.livegeekordie.com/article/nintendos-super-smashing-sale-week-4/

I personally prefer the http one but is it a CSS issue I'm guessing?

**Comment by Murphinator2000** [comment]
If you click the link make sure you put https:// in front as I think it defaults to http://

**Comment by jaredhaight** [comment]
Correct, the CSS and potentially JS won't load over https. While it's a
poor design, the https links aren't actually intended to be shared with
anyone. We're using https purely to protect user credentials.

In addition to the dependencies not loading over HTTPS, browsing the site
also skips our caching layer. Another reason why these links should not be
shared out through facebook or what have you.

I may be able to build some logic that only allows logged in users to
browse over https, and will redirect others to an http version of the page
if available.
